server-spi/src/main/java/org/keycloak/provider/ProviderConfigProperty.java

```diff
@@ -57,6 +57,11 @@
      */
     public static final String TEXT_TYPE="Text";
 
+    /**
+     * Define multiple claims as a Map of (claim, value) pairs
+     */
+    public static final String CLAIMS_TYPE="Claims";
```

Comment:
This should be changed to Map, so in can be re-used for other use-cases. End of the day it's just key-value mapping which can be useful elsewhere right?

Reply:
Yes right now this can be a general map. However if we want to build more complex boolean expressions with multiple claims (see our discussion in the mailing list https://lists.jboss.org/pipermail/keycloak-dev/2019-September/012446.html) we need to change that type again or introduce a new one. For now I guess it's a good idea to have a map type that can be reused. I'll change that.

Reply:
Done, the type map can be used for any purpose now, not only for claims.